### ISSUE TITLE
Remove Uppy fork and use upstream implementation

### ISSUE THREAD

**Issue by JammingBen** [owncloud/web#10868]
We needed to fork Uppy in the past to make the cloud importer work. Since then we've been trying to get the necessary changes upstream to be able to remove the fork again. Especially now that a new major version seems to be around the corner, we should keep that in mind. This would also unblock https://github.com/owncloud/web/issues/9472.

Required changes:

* https://github.com/transloadit/uppy/pull/4561
* https://github.com/transloadit/uppy/pull/4556
* https://github.com/transloadit/uppy/pull/4554
* https://github.com/transloadit/uppy/pull/4553
* https://github.com/transloadit/uppy/pull/4596
* https://github.com/transloadit/uppy/pull/4555 (seems to be superseded by https://github.com/transloadit/uppy/pull/4619)
* https://github.com/transloadit/uppy/pull/4551 (seems to be superseded by https://github.com/transloadit/uppy/pull/4619)
* https://github.com/transloadit/uppy/pull/4621
